kpi/models/object_permission.py

```diff
@@ -918,7 +920,7 @@ def has_perm(self, user_obj, perm):
 
     @transaction.atomic
     def remove_perm(self, user_obj, perm, defer_recalc=False, skip_kc=False):
-        r"""
+        """
```

Comment:
I don't know why this was a raw string if it didn't have any backslashes, but just in case your editor or linter is doing this blindly, `r"""` is allowed under some (different) circumstances by PEP 257:
> Use r"""raw triple double quotes""" if you use any backslashes in your docstrings.

Reply:
👍